content/kotlin/concepts/math-methods/terms/sin/sin.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'Math.sin()'
```

Comment:
```suggestion
Title: 'sin()'
```

Reply:
This entry has many of the same issues as the other math submissions you've completed. Please look at the previously reviewed entries and make the required updates.